lib/core_dom/view.dart

```diff
@@ -72,7 +72,13 @@ class ViewPort {
     scope.rootScope.domWrite(() {
       dom.Node previousNode = _lastNode(insertAfter);
       _viewsInsertAfter(view, insertAfter);
-      _animate.insert(view.nodes, placeholder.parentNode, insertBefore: previousNode.nextNode);
+
+      // The condition is false when we use the emulated shadom DOM, and the view port
```

Comment:
Try write a failing test. We might already have some failing tests but not running them with emulated shadow dom - https://travis-ci.org/angular/angular.dart/jobs/106388373